t5x/te_helper.py

```diff
@@ -177,8 +178,6 @@ def extend_logical_axis_rules(rules):
   @staticmethod
   def update_fp8_metas(grad_accum, flax_mutables):
     update_coll = te.update_collections(grad_accum, flax_mutables)
-    # As the suggestion of FP8 training, updating FP8 scales as frequent as possible.
-    update_coll = te.update_fp8_metas(update_coll)
```

Comment:
Is there a way to detect that we have a good pair of t5x/te ?
Just to help with the transition. Only if it is easy to do.

Reply:
We can keep the invokation of `te.update_fp8_metas` for a while. The overhead should be ignorable based on previos experinece. After both T5X and TE are stable enough with the recent changes, then we can remove this. Does this make sense?

Reply:
What happens if we call it twice?
Otherwise, it make sense.